releasenotes/notes/dc-dynamical-decoupling-measurement-change-64fc66f3c2ac38f3.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Match the adjustment of measurements for devices with odd short gate lentghs.
```

Comment:
```suggestion
    Match the adjustment of measurements for devices with odd short gate lengths.
```

Reply:
Changed in https://github.com/Qiskit/qiskit-ibm-provider/pull/576/commits/1dce8d05e42369d47acd12ec3a1227c436e3d8b8